### DISCUSSION TITLE
names/titles in library (searching)

### DISCUSSION THREAD

**Discussion by pihentagy** [excalidraw/excalidraw#4241] [Q&A]
Can I add names to items in the library? It would be super cool…
Then I will able to search my library by title.

**Comment by dwelle** [comment]
Shortly, we will be adding a requirement to name your library items on publishing. After that, we'll add support for search locally.

So...

![image](https://user-images.githubusercontent.com/5153846/141614332-9a9ebb3b-7e8e-4b1d-ab72-82982c432913.png)

**Comment by pihentagy** [comment]
And I guess the one, who just downloads the library can also rename the downloaded items locally.

**Reply by dwelle** [reply]
Not initially. But not too long after.
